CONTRIBUTING.md

```diff
@@ -102,15 +102,31 @@ Detailed explanation:
 
 8. [Open a Pull Request](https://help.github.com/articles/using-pull-requests/)
     with a clear title and description.
+    
+    If page is not about standard Unix/Linux tool, please include link to the tool home page.
+    
+    If you are changing something non trivial, not just add page for new tool, please describe why are you doing this.
```

Comment:
"non-trivial"; "not just add" --> "not just adding"; "why are you" --> "why you are" (it's not a question, so no need to reverse the order of the words)